### DISCUSSION TITLE
Default custom email booking from waiting list

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#642] [General]
When taking a booking from the waiting list, you can choose whether an email should be sent concerning the booking changes. 
Changes can be anything, like number of tickets, add information, etc. But removing someone from the waitinglist is a regular action.
Is it possible to select a default template for this action?

**Comment by liedekef** [comment]
I'm not sure I understand correctly, but when a booking is removed from the waiting list it goes into "pending" status. So the mail that is sent out is the pending mail-template, so you can configure that.

**Comment by JanineKall** [comment]
Yep I know, but what if I want to send another mail when I remove someone from the waiting list by default?

**Reply by liedekef** [reply]
ok, I see. The next version will have 4 extra filters, so you can filter (=change) the pending booking text yourself:

eme_rsvp_email_from_pending_to_waitinglist_subject
eme_rsvp_email_from_pending_to_waitinglist_body
eme_rsvp_email_from_waitinglist_to_pending_subject
eme_rsvp_email_from_waitinglist_to_pending_body

By default they have the same value as the pending mail/subject.
See https://github.com/liedekef/events-made-easy/commit/f08d5398dcc288dfa6a85b295dd9f4eb4454b3f8

**Reply by JanineKall** [reply]
Ok that is certainly worth waiting for
